src/textual/css/tokenizer.py

```diff
@@ -148,8 +149,8 @@ def __rich_repr__(self) -> rich.repr.Result:
 
 
 class Tokenizer:
-    def __init__(self, text: str, path: str = "") -> None:
-        self.path = path
+    def __init__(self, text: str, path: str | PurePath = "") -> None:
+        self.path = str(path)
```

Comment:
in order to avoid too much changes with the introduction of PathLike objects, I cast this `path` to a string so the behaviour of the Tokenizer is still the same